web3/contract.py

```diff
@@ -268,6 +287,52 @@ def deploy(cls, transaction=None, args=None, kwargs=None):
         txn_hash = cls.web3.eth.sendTransaction(deploy_transaction)
         return txn_hash
 
+    @classmethod
+    @deprecated_for("contract.<constructor>.buildTransaction")
+    def deploy_data(cls, transaction=None, args=None, kwargs=None):
```

Comment:
Can we remove this method since it was never part of the original API.

Reply:
Yes